src/FINDOLOGIC/Helpers/SendRequestHelper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace FINDOLOGIC\Helpers;
+
+use FINDOLOGIC\Exceptions\ServiceNotAliveException;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+
+class SendRequestHelper
```

Comment:
An idea: Create a `FindologicClient`, receiving the service ID on construction, with methods like `isAlive`, `search`, `navigate`, and `suggest`.